src/js/sdk.ts

```diff
@@ -100,14 +100,16 @@ export function init(
 }
 
 /**
- * Sets the release on the event.
+ * Legacy. Sets the release on the event.
```

Comment:
L: is adding a doc comment the only way of making it legacy/deprecated? in Java, we have the `@Deprecated` annotation.

Reply:
There is the `@deprecated` tag, but I couldn't find anything about a `@legacy` tag

Reply:
mm but what's the meaning of legacy here? do we intend to remove it in the future in favor of options.setRelease or what? because if legacy means, we are keeping it for compatibility but there's another way to do it, I see that being `@deprecated` then

Reply:
Hmm I agree then, I'll change it to `@deprecated`!